lib/core/src/client/preview/client_api.js

```diff
@@ -95,7 +95,7 @@ export default class ClientApi {
       }
 
       if (this._storyStore.hasStory(kind, storyName)) {
-        throw new Error(`Story of "${kind}" named "${storyName}" already exists`);
+        logger.warn(`Story of "${kind}" named "${storyName}" already exists`);
```

Comment:
Basically, I agree with this change, though I wonder whether we want to replace the existing story with this `kind`
@tmeasday you've worked a lot in this area. WDYT ?